src/python/grpcio/grpc/_cython/_cygrpc/common.pyx.pxi

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class BaseError(Exception):
+    """
+    This class is EXPERIMENTAL.
+    The base class for exceptions generated by gRPC.
+    """
+
+class UsageError(BaseError):
+    """
+    This class is EXPERIMENTAL.
```

Comment:
Can we make sure that these classes and docstrings end up in the generated documentation? I'm not sure if it will show up since 1) it's included via reimport and 2) it's defined in Cython.

Reply:
We should have them, before this change, those errors are also defined in Cython and included via reimport in Async stack, and they're included in our documentation: https://grpc.github.io/grpc/python/grpc_asyncio.html#grpc-exceptions